public/app/features/alerting/unified/components/rules/central-state-history/CentralAlertHistorySceneExposedComponent.tsx

```diff
@@ -0,0 +1,13 @@
+import { Suspense, lazy } from 'react';
+
+import { CentralAlertHistorySceneProps } from '../../../../../../../../packages/grafana-data/src/types/pluginExtensions';
+
```

Comment:
nit: `CentralAlertHistorySceneProps` needs to be exported from `@grafana/data` so it can be imported without using relative paths. We would like to end with the following import:

```suggestion
import { CentralAlertHistorySceneProps } from '@grafana/data';

```